tests/python/common/ops/test_ops.py

```diff
@@ -413,14 +413,14 @@ def test_gather_mm_idx_b(feat_size, dtype, tol):
         and not torch.cuda.is_bf16_supported()
     ):
         pytest.skip("BF16 is not supported.")
-
-    if (
-        F._default_context_str == "gpu"
-        and dtype == torch.float16
+    if F._default_context_str == "gpu" and (
+        dtype == torch.float16
         and torch.cuda.get_device_capability() < (7, 0)
+        or dtype == torch.bfloat16
+        and torch.cuda.get_device_capability() < (8, 0)
```

Comment:
Sorry I am not good at understanding the priority of `or` and `and`, can you add parentheses?

Reply:
Done